source/administration/batch-framework.rst

```diff
@@ -0,0 +1,153 @@
+.. _minio-batch-framework:
+
+===============
+Batch Framework
+===============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. versionchanged:: MinIO RELEASE.2022-10-08T20-11-00Z or later
+
+   The Batch Framework was introduced with the ``replicate`` job type in the :mc:`mc` RELEASES.2022-10-08T20-11-00Z.
```

Comment:
You can probably just have `.. versionadded::` up top and leave it at that.

Reply:
Wait, you're telling me we have `.. versionchanged::` AND `.. versionadded::`?

Fixed.